site/docs/v1/tech/developer-guide/api-gateways/aws-api-gateway.adoc

```diff
@@ -0,0 +1,217 @@
+--header 'Authorization: <API_KEY>' \
+--header 'Content-Type: application/json' \
+--data-raw '  {
+    "loginId": "<USER_EMAIL>",
+    "password": "<USER_PASSWORD>",
+    "applicationId": "<APPLICATION_ID>",
+    "noJWT" : false
+  }'
+----
+
+Where:
+
+* `<YOUR_FUSIONAUTH_URL>` is the URL of your FusionAuth installation.
+* `<API_KEY>` is the API created above, enabled for login.
+* `<USER_EMAIL>` is the email address of the user added above.
+* `<USER_PASSWORD>` is the password of the user added above.
+
+The return response from FusionAuth should look similar to the following:
+
+[source,json,title="Return token from the API call"]
+----
+{
+    "token": "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6ImhDUjA4X3daR2s0OUFlYUFmRDY5ZmJKWmRGTSJ9.eyJhdWQiOiI2M2I3M2Y3Ni03NDAwLTQ4N2QtYjEyMi01NzA1Yjg0OGRhODAiLCJleHAiOjE2NzMzNjYyMDQsImlhdCI6MTY3MzM2MjYwNCwiaXNzIjoiaHR0cHM6Ly9mdXNpb25hdXRoLnJpdHphLmNvIiwic3ViIjoiMzk2MzAwMGYtNjg2ZC00MTY5LWI2MjgtOWM5YzQ1MzRiNzgwIiwianRpIjoiZDk3ZGIyZWYtZjExNS00ZDIxLWFlOTQtMDIyN2RmMGU4YzI5IiwiYXV0aGVudGljYXRpb25UeXBlIjoiUEFTU1dPUkQiLCJlbWFpbCI6ImJvYkBhd3MuY29tIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsInByZWZlcnJlZF91c2VybmFtZSI6ImJvYmF3cyIsImFwcGxpY2F0aW9uSWQiOiI2M2I3M2Y3Ni03NDAwLTQ4N2QtYjEyMi01NzA1Yjg0OGRhODAiLCJyb2xlcyI6W10sImF1dGhfdGltZSI6MTY3MzM2MjYwNCwidGlkIjoiZjAwNGMxZmUtNDg0Yi05MDJjLWQ3Y2EtYmRiYzQ2NGRhMGI3In0.m7gzXhNLToPNVE1p5Vo2pLgP6WBcPNfS_zZJnJ81mdEgi6-orViz-tU8j0L8wva0-8KlMdy54cq_XjnDnYJ0aX90O4ZE_QVU5NuDDfzXH14wQtKQoIIydsB6ZvQoBt8JNFUHJb9ANLCGnfn6FVQKqPIzye18Gx_7wYSVokw3eLNFyzrq9dwOD5Q8V9gvZmXV2pTokQAtA7qFaadb2dIeFlSEB7wamKiZLXILjeWAeMbbvAAMQZWFh46UJjwr06QTd8PxQmRwDWWznJy1Vs8EAgZA4vkRSWnn3IbiaCtOaL1ANuEex6il7q32ahxj0Ncm9wn0DbDsQE9NB0CCNTSIhA",
+    "tokenExpirationInstant": 1673366204805,
+    "user": {
+        ....
+    }
+}
+----
+
+Copy the `token` value out. This is the JWT, which we can use to access the API gateway function.
+
+Now call the API gateway with the following cURL command:
+
+[source,shell,title="Calling the API Gateway endpoint with the JWT"]
+----
+curl --location --request GET 'https://<GATEWAY_URL>/privateFunction' \
+--header 'Authorization: Bearer <JWT_TOKEN>'
+----
+
+Where:
+
+* `<GATEWAY_URL>` is the invoke URL saved earlier.
+* `<JWT_TOKEN>` is the JWT from the `/api/login` call.
+
+You should see the function return with a reflection of the claims it received:
+
+[source,json,title="Return token from the API call"]
+----
+{
+    "applicationId":"63b73f76-7400-487d-b122-5705b848da80",
+    "aud":"63b73f76-7400-487d-b122-5705b848da80",
+    "auth_time":"1672137288",
+    "authenticationType":"PASSWORD",
+    "email":"ehrlich@piedpiper.com",
+    "email_verified":"true",
+    "exp":"1672140888",
+    "iat":"1672137288",
+    "iss":"https://local.fusionauth.io",
+    "jti":"2f80b975-1870-4970-b876-90c2e7d9444b",
+    "preferred_username":"erlich",
+    "roles":"[]",
+    "sub":"3963000f-686d-4169-b628-9c9c4534b780",
+    "tid":"f004c1fe-484b-902c-d7ca-bdbc464da0b7"
+ }
+----
+
```

Comment:
Should add a troubleshooting section.

This won't work with a local installation, because the API gateway needs to be able to access the JWKS.json to validate the key.

If you don't set up asymmetric keys, it will fail. The key must be an RSA key as well.

If you wait too long between when you run the login API and when you call the API and the token expires it will fail.